src/cljs/ui/new_analysis/discrete_mcc_tree.cljs

```diff
@@ -32,7 +34,7 @@
 
 (defn discrete-mcc-tree []
   (let [discrete-mcc-tree (re-frame/subscribe [::subs/discrete-mcc-tree])
-        field-errors      (re-frame/subscribe [::subs/discrete-mcc-tree-field-errors])]
+        field-errors      (r/atom nil)]
```

Comment:
Why choose ratom over a subscription? This will make error go away if component is unmounted (user goes to a different tab for example)

Reply:
Yeah so I tend to go with local state for things that aren't going to be accessed from outside the component so we don't end up polluting with events, subscriptions, db spec, etc. I don't think people expect to be filling a form and then jump into a random page and come back to keep filling the form, I don't think is how this kind of app normally works.

Reply:
OK.